test/cpp/qps/client_sync.cc

```diff
@@ -170,8 +172,16 @@ class SynchronousStreamingClient final : public SynchronousClient {
     if (stream_[thread_idx]->Write(request_) &&
         stream_[thread_idx]->Read(&responses_[thread_idx])) {
       entry->set_value((UsageTimer::Now() - start) * 1e9);
+      // don't set the status since there isn't one yet
       return true;
     }
+    Status s = stream_[thread_idx]->Finish();
+    // don't set the value since the stream is failed and shouldn't be timed
+    entry->set_status(s.error_code());
+    if (!s.ok()) {
+      gpr_log(GPR_ERROR, "Stream %zu received an error %s", thread_idx,
```

Comment:
I think windows doesn't support `%zu`. We usually end up casting `size_t` to `unsigned long`. :/